tests/props/host/host.c

```diff
@@ -8,8 +8,51 @@
 #include <openenclave/internal/tests.h>
 #include <stdio.h>
 #include "../../../host/sgx/enclave.h"
+#include "../host/sgx/cpuid.h"
 #include "props_u.h"
 
+#ifdef oe_get_xfrm
+#undef oe_get_xfrm
+#endif
+
+bool avx_supported()
```

Comment:
Style: Prefix local static functions with `_` and start functions with verbs so `_is_avx_supported()`

Reply:
This wasn't static function earlier but made it static and changed the name. Done.